jetbrains-core/src/software/aws/toolkits/jetbrains/components/telemetry/LoggingDialogWrapper.kt

```diff
@@ -0,0 +1,72 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.components.telemetry
+
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.ui.DialogWrapper
+import software.aws.toolkits.core.telemetry.TelemetryNamespace
+import software.aws.toolkits.jetbrains.services.telemetry.TelemetryService
+import java.awt.Component
+
+// DialogWrapper constructor signatures:
+//  protected DialogWrapper(@Nullable Project project, boolean canBeParent)
+//    this(project, canBeParent, IdeModalityType.IDE);
+//  }
+//  protected DialogWrapper(@Nullable Project project, boolean canBeParent, @NotNull IdeModalityType ideModalityType)
+//    this(project, null, canBeParent, ideModalityType);
+//  }
+//  protected DialogWrapper(@Nullable Project project, @Nullable Component parentComponent, boolean canBeParent, @NotNull IdeModalityType ideModalityType) {
+//    <logic>
+//  }
+//  protected DialogWrapper(@Nullable Project project) {
+//    this(project, true);
+//  }
+//  protected DialogWrapper(boolean canBeParent) {
+//    this((Project)null, canBeParent);
+//  }
+//  @Deprecated
+//  protected DialogWrapper(boolean canBeParent, boolean applicationModalIfPossible)
+//    this(null, canBeParent, applicationModalIfPossible);
+//  }
+//  protected DialogWrapper(Project project, boolean canBeParent, boolean applicationModalIfPossible) {
+//    <logic>
+//  }
+//  protected DialogWrapper(@NotNull Component parent, boolean canBeParent) {
+//    <logic>
+//  }
```

Comment:
Do we need all these comments? Doesn't really seem very useful to me.

Reply:
I don't mind removing them. It's just easier to tell what the original constructors are when you don't have to try to find them in the parent's source code

Reply:
The constructors are just a set of different forms for supporting different parameter signatures. It is pretty strait forward I think. I don't mind leaving it there, it's just doesn't standard docs :)